data/ecosystems/s/seascape-network.toml

```diff
@@ -30,78 +24,6 @@ url = "https://github.com/blocklords/blocklords2d-smartcontracts"
 [[repo]]
 url = "https://github.com/blocklords/claim-crowns"
 
-[[repo]]
-url = "https://github.com/blocklords/contract-interactor"
-
-[[repo]]
-url = "https://github.com/blocklords/crowns"
-
-[[repo]]
-url = "https://github.com/blocklords/crowns-wrapper"
-
-[[repo]]
-url = "https://github.com/blocklords/gosds"
-
-[[repo]]
-url = "https://github.com/blocklords/ice-bridge"
-
-[[repo]]
-url = "https://github.com/blocklords/meter-bridge-token"
-
-[[repo]]
-url = "https://github.com/blocklords/miner-smartcontracts"
-
-[[repo]]
-url = "https://github.com/blocklords/moonscape-website"
-
-[[repo]]
-url = "https://github.com/blocklords/nft-brawl-leaderboard"
-
-[[repo]]
-url = "https://github.com/blocklords/nft-brawl-leaderboard-payout"
-
-[[repo]]
-url = "https://github.com/blocklords/opensea-metadata"
-
-[[repo]]
-url = "https://github.com/blocklords/redis-limiter"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-adapter-interface"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-cli"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-evm-adapter"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-hardhat"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-truffle"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-ts"
```

Comment:
a lot of these were reassigned to https://github.com/seascapenetwork/... org, could you re-add them with the new owner instead please?